include/aspect/material_model/utilities.h

```diff
@@ -41,6 +41,161 @@ namespace aspect
      */
     namespace MaterialUtilities
     {
+      namespace Lookup
+      {
+
+
+
+        /**
+         * A base class that can be used to look up material data from an external
+         * data source (e.g. a table in a file). The class consists of data members
+         * and functions to access this data, but it does not contain the functions
+         * to read this data, which has to be implemented in a derived class.
+         */
+        class MaterialLookup
+        {
+          public:
+
+            double
+            specific_heat(double temperature,
+                          double pressure) const;
```

Comment:
not your fault, but change the two `double` arguments to `const double`. Same in the function below.